### ISSUE TITLE
Implementation of Distilling Causal Effect

### ISSUE THREAD

**Issue by lichong952012** [JoyHuYY1412/DDE_CIL#5]
Thanks for your excellent work. I have a question, in equation 6 of the article, how is the multiplication P (Y|I= i) *Wi calculated in the code? In loss1, the weight W is not included.

**Comment by JoyHuYY1412** [comment]
> Thanks for your excellent work. I have a question, in equation 6 of the article, how is the multiplication P (Y|I= i) *Wi calculated in the code? In loss1, the weight W is not included.

@lichong952012 Hi, please check here 
https://github.com/JoyHuYY1412/DDE_CIL/blob/d0820b55961ec83f58ba3c8b5a57a79a1bf38062/utils_incremental/incremental_train_and_eval_DCE_MR_LF_TDE.py#L224

The logits are weighted summed.
